schema.json

```diff
@@ -722,11 +853,34 @@
             "type": "string"
           }
         },
+        "required": [
+          "name",
+          "position",
+          "startDate",
+          "endDate"
+        ],
         "type": "object"
       },
       "type": "array"
     }
   },
+  "required": [
```

Comment:
Does it mean that we require the section to be there, but empty or actually be filled?
If it can be empty, I agree, if not that feels problematic as not everyone might have certificates for example.

Reply:
I think the sections should be provided, but a user, for example, does not any certificate it should just be an empty array.

Reply:
Ok great. So section there, but empty fulfills the requirement. That works.